website/client-old/js/controllers/authCtrl.js

```diff
@@ -11,6 +11,16 @@ angular.module('habitrpg')
 
       $scope.logout = function() {
         localStorage.clear();
+
```

Comment:
Add another comment here saying that this deletes cookies, except for those that have the HttpOnly flag set.

Since this code is client side, it can't delete HttpOnly cookies, but we'll just have to live with that.